influxql/parser.go

```diff
@@ -577,6 +577,15 @@ func (p *Parser) parseSelectStatement(tr targetRequirement) (*SelectStatement, e
 		return nil, err
 	}
 
+	// Set if the query is a raw data query or one with an aggregate
+	stmt.IsRawQuery = true
```

Comment:
OK, but doesn't your second example still result in two substatements within the code? One for `host` and one for `min(value)`. It's been a while since I looked at the relevant code, I just recall that "substatement" concept and want to check that it's all OK.